packages/client/env.ts

```diff
@@ -7,7 +7,16 @@ import _configs from '/@slidev/configs'
 export const configs = _configs as SlidevConfig
 export const slideAspect = configs.aspectRatio ?? (16 / 9)
 export const slideWidth = configs.canvasWidth ?? 980
-export const slideHeight = Math.round(slideWidth / slideAspect)
+
+// The height must be an integer (apply round) and it should honors the aspect ratio more as possible.
+// Also, to prevent errors on PDF export, we need to increase the height by 1px at time until the real
+// aspect ratio is equal or less than the requested one.
+// Doing this, we will prevent to create empty white pages after all single pages.
+let calculatedHeight = Math.round(slideWidth / slideAspect)
+while (slideAspect < (slideWidth / calculatedHeight))
+  calculatedHeight++
+
+export const slideHeight = calculatedHeight
```

Comment:
Do you mean `Math.ceil`?

Reply:
`calculatedHeight` is already an integer since at line 15 is calculated from Math.round. I'm wrong?

Reply:
I mean we can change `Math.round` to `Math.ceil`

Reply:
Changes done, let me know 😉 

Also: since I'm already in, can I ask to enable this PR for the Hacktoberfest?

Reply:
Sorry for the message @antfu: what about to make this PR enabled for Hacktoberfest?